src/controllers/bookingRequest/approveBookingRequestIntent.controller.js

```diff
@@ -0,0 +1,98 @@
+const { OK } = require("http-status");
+const BookingRequest = require("../../models/bookingRequest.model");
+const {
+  getConflictingBookingRequests,
+} = require("../../services/bookingRequest.service");
+const { convertUnixToDateString } = require("../../utils/dateToUnix");
+const { mapSlotsToTiming } = require("../../utils/mapSlotsToTiming");
+
+// returns what requests will be rejected on approval
+
+const approveBookingRequestIntentController = async (req, res, next) => {
+  const query = req.query;
+  const bookingRequestId = query.bookingRequestId;
+
+  let bookingRequest;
+  try {
+    bookingRequest = await BookingRequest.findOne({
+      _id: bookingRequestId,
+    }).populate("venue");
+  } catch (err) {
+    return next(err);
+  }
+
+  if (!bookingRequest) {
+    const message = "Invalid booking requestId";
+    const err = errorFormatter(message, BAD_REQUEST);
+    return next(err);
+  }
+
+  const email = bookingRequest.email;
```

Comment:
line 30-36 all these not needed right, i think u didnt use these constants in your code